source/components/triggering-changes-with-actions.md

```diff
@@ -156,3 +156,56 @@ a function instead.
 Actions in components allow you to
 decouple an event happening from how it's handled, leading to modular,
 more reusable components.
+
+## Calling Actions Across Component Layers
```

Comment:
"Across" might be a bit ambiguous as people can read it as siblings instead of nested.